src/app/validation/config/config.exs

```diff
@@ -0,0 +1,18 @@
+import Config
+
+# BIG ISSUE with this method: private key gets fed in as start_link args to goth, so it shows up in sasl logs
+# TODO: custom credentials module
```

Comment:
No, but I updated the configuration so that the goth config can be provided via a `.gitignore`'d file, so that devs can just drop their key in there directly. CI doesn't need to have google cloud credentials to run static analysis in tests, so we can just leave it unconfigured there.